contribution_to_project.rst

```diff
@@ -0,0 +1,95 @@
+***********************
+Contributing to Ansible
+***********************
+
+This document describes ways how you can contribute to the Open Source projects of the Ansible ecosystem.
+
+.. contents:: Topics
```

Comment:
Our style guide suggests no heading for a local TOC - see https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#adding-local-tocs.
```suggestion
.. contents::
   :local:
```